examples/client.c

```diff
@@ -1015,8 +1023,17 @@ get_session(
 	coap_address_init( &bind_addr );
 	bind_addr.size = rp->ai_addrlen;
 	memcpy( &bind_addr.addr, rp->ai_addr, rp->ai_addrlen );
-	if ( identity && key && (proto == COAP_PROTO_DTLS || proto == COAP_PROTO_TLS) )
-	  session = coap_new_client_session_psk( ctx, &bind_addr, dst, proto, identity, key, key_len );
+        if (cert_file && (proto == COAP_PROTO_DTLS || proto == COAP_PROTO_TLS)) {
+          coap_dtls_pki_t dtls_pki;
+          memset (&dtls_pki, 0, sizeof(dtls_pki));
+          dtls_pki.public_cert = cert_file;
+          dtls_pki.private_key = cert_file;
+          session = coap_new_client_session_pki(ctx, &bind_addr, dst, proto, &dtls_pki);
+        }
+        else if ((identity || key) &&
```

Comment:
For better readability, I would like to have parentheses about the if-then-else cases.

Reply:
Just updated the code